Assets/MixedRealityToolkit-SDK/Features/Input/GazeProvider.cs

```diff
@@ -234,7 +235,7 @@ public override bool TryGetPointerRotation(out Quaternion rotation)
             /// <param name="handedness">Optional handedness of the source that pressed the pointer.</param>
             public void RaisePointerDown(MixedRealityInputAction mixedRealityInputAction, Handedness handedness = Handedness.None)
             {
-                InputSystem.RaisePointerDown(this, handedness, mixedRealityInputAction);
+                MixedRealityManager.InputSystem.RaisePointerDown(this, handedness, mixedRealityInputAction);
```

Comment:
Not checking for no Input system
```
MixedRealityManager.InputSystem?.
```
Update and check throughout

Reply:
We wouldn't get this event called without an input system in the first place. Null check is redundant.